### PR TITLE
Add indri

### PR DESCRIPTION
**Pull request by ghost**
Import from upstream/oracle/userland-gate

### CODE REVIEW

File: components/dnsmasq/Makefile [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+COMPONENT_BUILD_ARGS += CFLAGS="$(CFLAGS)"
+COMPONENT_BUILD_ARGS += PREFIX="/usr"
+COMPONENT_BUILD_ARGS += LOCALEDIR="/usr/share/locale"
+COMPONENT_BUILD_ARGS += LDFLAGS="$(LDFLAGS) -m64"
+COMPONENT_BUILD_TARGETS = "all-i18n"
+COMPONENT_INSTALL_ARGS += PREFIX="/usr"
+COMPONENT_INSTALL_TARGETS = "install-i18n"
+
+# common targets
+build:		$(BUILD_64)
```
Comment (pyhalov) [inline components/dnsmasq/Makefile:56]: Component should be 32bit  or (32bit and 64bit) if there are no strong reasons for it to be not. You can use isaexec to make startup scripts simple.

**Comment by alarcher** [comment]
Should we just add 32bit builds ?

**Comment by pyhalov** [comment]
why not? We still claim that we support 32-bit sytems. And I hope dnsmasq doesn't require 64bit address space

**Comment by alarcher** [comment]
I meant that if it is OK I can update the branch to build 32bit only and be able to merge. ;)

**Comment by pyhalov** [comment]
It's OK

**Comment by alarcher** [comment]
This PR can be closed as changes are duplicated in #1257 and #1258.

**Pull request closed without merging**